examples/reactnative/rn_0.59/android/app/build.gradle

```diff
@@ -77,6 +77,7 @@ project.ext.react = [
 ]
 
 apply from: "../../node_modules/react-native/react.gradle"
+apply from: "../../node_modules/@bugsnag/react-native/react.gradle"
```

Comment:
Is the name `react.gradle` a bit misleading? Perhaps we could go with `bugsnag.gradle`, `bugsnag-react-native.gradle` or `build.gradle`.

Reply:
happy with any of those - any particular favourite?